timed/conftest.py

```diff
@@ -14,8 +14,8 @@
 
 
 def register_module(module):
-    for name, obj in inspect.getmembers(module):
-        if isinstance(obj, FactoryMetaClass) and not obj._meta.abstract:
+    for _name, obj in inspect.getmembers(module):
```

Comment:
Is `name` a reserved word now? or future reserved word? IMO there's no reason to rename this var

Reply:
AFAIK no, but its unused and ruff doesn't like that, an alternative would be `_`

Reply:
Ah right, then leave it as is, my bad